python/cudf/cudf/core/indexed_frame.py

```diff
@@ -2608,12 +2608,15 @@ def sort_index(
                     and self._data.multiindex
                 ):
                     out._set_column_names_like(self)
+            if ignore_index is True:
```

Comment:
This is supposed to be a `bool`, so don't use `is` for boolean comparison.
```suggestion
            if ignore_index:
```